go-tipb/executor.pb.go

```diff
@@ -1061,10 +1061,12 @@ func (m *Aggregation) GetChild() *Executor {
 
 type TopN struct {
 	// Order by clause.
-	OrderBy          []*ByItem `protobuf:"bytes,1,rep,name=order_by,json=orderBy" json:"order_by,omitempty"`
-	Limit            uint64    `protobuf:"varint,2,opt,name=limit" json:"limit"`
-	Child            *Executor `protobuf:"bytes,3,opt,name=child" json:"child,omitempty"`
-	XXX_unrecognized []byte    `json:"-"`
+	OrderBy            []*ByItem `protobuf:"bytes,1,rep,name=order_by,json=orderBy" json:"order_by,omitempty"`
+	Limit              uint64    `protobuf:"varint,2,opt,name=limit" json:"limit"`
+	PartitionBy        []*ByItem `protobuf:"bytes,3,rep,name=partition_by,json=partitionBy" json:"partition_by,omitempty"`
+	RecommendedPartNum *uint64   `protobuf:"varint,4,opt,name=recommended_part_num,json=recommendedPartNum" json:"recommended_part_num,omitempty"`
```

Comment:
Same comment on removing this since it is needed for planned change with only k=0.

Reply:
Oh, all the change of `xxx.pb.go` is stub code generated by GRPC. I just modify the `.proto` and execute `make go` then it generates these diff. And the CI will generate it again and compare it to ensure the change is valid.
So usually we just need to pay attention to the proto change.